lib/IDE/SyntaxModel.cpp

```diff
@@ -1140,7 +1155,8 @@ bool ModelASTWalker::shouldPassBraceStructureNode(BraceStmt *S) {
   return (!dyn_cast_or_null<AbstractFunctionDecl>(Parent.getAsDecl()) &&
           !dyn_cast_or_null<TopLevelCodeDecl>(Parent.getAsDecl()) &&
           !dyn_cast_or_null<CaseStmt>(Parent.getAsStmt()) &&
-          S->getSourceRange().isValid());
+          S->getSourceRange().isValid() &&
+          !S->isImplicit());
```

Comment:
I think this change makes sense, but with my understanding, this doesn't affect this specific case because it's skipped anyway?

Reply:
Yeah, this was what I did to fix the initial assertion hit. The special handling of interpolated string literal exprs (which I did to stop emitting call structure nodes on each interpolation when we didn't previously) ended up making it redundant. I left it because I figured it would help if us do the right thing if TapExpr ever ends up being used in other places, but we can remove it if it turns out the BraceStmt should not be implicit.